plugins/samson_ledger/test/lib/samson_ledger/client_test.rb

```diff
@@ -29,10 +40,71 @@
   end
 
   describe ".post_deployment" do
+    before do
+      @asserts = Proc.new {}
```

Comment:
they normally get removed after a run ... but segfault will not run ensure blocks ... so they stay